website/content/en/docs/User Guide/Learn Ops Management/cloud_slo_monitoring.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "SLIs, SLOs and Burn rate Alerts"
+linkTitle: "SLIs, SLOs and Burn rate Alerts"
+weight: 40
+---
+
+
+{{% pageinfo %}}
+* [Overview](#SLIs,-SLOs-and-Burn-rate-Alerts-Overview)
+* [Services SLOs](#Services-SLOs)
+* [Configure your own SLIs and SLOs](#Configure-your-own-SLIs-and-SLOs)
+* [Configure Burn Rate Alerts](#Configure-Burn-Rate-Alerts)
+{{% /pageinfo %}}
+
+
+#### SLIs, SLOs and Burn rate Alerts Overview 
+Cloud Operations Sandbox comes with several predefined SLOs (Service level objectives), that allow you to measure your users happiness. To learn more about SLIs and SLOs [SRE fundamentals.](https://cloud.google.com/blog/products/devops-sre/sre-fundamentals-slis-slas-and-slos)
+
+Cloud operations suite provides **service oriented monitoring**, that means that you are configuring SLIs, SLOs and Burning Rates Alerts for a 'service'.  
+
+The first step in order to create SLO is to **ingest the data**. For GKE services telemetry and dashboards comes out of the box, but you can also ingest additional data and [create custom metrics.](#Monitoring-and-logs-based-metrics)
+
+Then you need to **define your service**, Cloud Operations Sandbox' services are already detected since Istio's services are automatically detected and created. But to demonstrate that you can create your own services, it also deploys custom services using [Terraform](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring).
+
+You can find all the services under [monitoring → services → Services Overview](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/ui/svc-overview), and you can create your own [custom service.](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/ui/define-svc)
+
+![image](/docs/images/user-guide/37-services-overview.png)
+#### Services SLOs  
+
+The *predefined SLOs* are also deployed as part of [Terraform code](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring/04_slos.tf) and currently are for the mentioned custom services, the Istio service and Rating service.  
+
+**Custom services SLOs**
+``` 
+Custom service availability SLO: 90% of HTTP requests are successful within the past 30 day windowed period
+```
```

Comment:
maybe you already do it, but if not - in Visual Studio Code there is markdown linter extension: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint If you open markdown in VSCode with this extension - it will point out all the issues with markdown and it's easy to fix.

Reply:
I'm using it, something specific in mind?